src/components/inputs/Pin/index.js

```diff
@@ -14,6 +14,7 @@ const Wrapper = styled.div`
 const StyledInput = styled.input`
     letter-spacing: 7px;
     width: 100%;
+    height: 53px;
```

Comment:
why not keep the height prop adjustable?

```height: ${props => (props.height ? `${props.height}px` : '40px')};```

Reply:
Because we want it to look the same everywhere.
If there will be a good reason why to do it, or there will be a request to do it we can make it. 
But i don't see now a reason why to change the height. (...or we can maybe add size prop like 'xs', 'md', 'lg')